data/wanderers middle.txt

```diff
@@ -3158,9 +3144,9 @@ event "wanderers: sabira eseskrai colony"
 		fleet "Wanderer Drones" 800
 		fleet "Kor Efret Home" 800
 	system "Furmeliki"
-		fleet "Large Quarg" 1000
-		fleet "Kor Efret Home" 800
+		add fleet "Kor Efret Home" 800
 	planet "Sabira Eseskrai"
+		add attributes wanderer
```

Comment:
This gives Sabira Eseskrai `attributes korath wanderer` which seems very appropriate given that the Efret and Wanderer populations are rehabilitating this world together. Without this line, Sabira remains `attributes korath`.